docs/src/use/configure/migration-guide.md

```diff
@@ -114,6 +114,83 @@ export default [
 ];
 ```
 
+### Processors
+
+In eslintrc files, processors had to be defined in a plugin, and then referenced by name in the configuration. Processors with a name in the special form `.xxx`  were also automatically configured for any files with the extension `xxx`.
```

Comment:
xxx has some connotations that aren't appropriate for documentation. :smile:

```suggestion
In eslintrc files, processors had to be defined in a plugin, and then referenced by name in the configuration. Processors beginning with a dot indicated a filename extension (such as `.md`) were also automatically configured for any files matching that extension.
```

Reply:
🤦 Whoops, didn't even register, good catch 😄

Reply:
Gone a step further and linked out to the custom processors page so that developers can be sure what we're talking about and get more details:

> Processors beginning with a dot indicated a [file extension-named processor](../../extend//custom-processors.md#file-extension-named-processor) which ESLint would automatically configure for that file extension.